### ISSUE TITLE
<aside>-tag in a template causes still conflict in testing-tool for accessibility 

### ISSUE THREAD

**Issue by grashalm4u** [BugBuster1701/contao-banner-bundle#82]
hello bugbuster,
I'm sorry if I open this issue again. you had my issue no. 68 closed in May 2022, and I don't know if I can or may reopen it, so a new issue to be on the safe side (I'm an absolute beginner in github).

In October 2022 I posted again that the error message still existed, but there was no further reaction.

Today I tested my website again for accessibility, but before that I left the original template "mod_banner_list_all.html5" from the current extension version 1.5.0 in it. unfortunately the test tool still displays the following error message: "aside should not be contained in another landmark." The program displays the following as an issue description: "Ensures the complementary landmark or aside is at top level". The element location is named: "#bmid370", and the element source is specified: "<aside id="bmid370" title="banner_bmid370" class="mod_banner block">".
the program advises: "Fix the following: The complementary landmark is contained in another landmark."

The tool I tested with this time is called: "axe DevTools v4.49.0" and is a free extension in firefox that can be called up with the DevTools after installation. tested according to WCAG 2.1 AA and I set the "Best Practices" button to "ON".
maybe you can easily reproduce this error message with it.

Last year I had used the axe DevTools as well, but I didn't mention them because I didn't want to make too much of a mess. I haven't used the "rocket validator" tool since then, but the axe DevTools seem to me to be completely sufficient here, even if they can only test a single page at a time. And they are easy to install and handle.

one suggestion from the forum at the former time was to replace the "aside" tags with "div" tags. With this template (see above) it works wonderfully and the error message no longer appears. I can't judge whether this has any other negative effect on the functioning of the template.
Unfortunately, due to lack of knowledge, I can't judge / assess whether this axe DevTool works reliably, but it makes a solid impression on me at first.

i would be very happy if you could take a moment to look at this story again and see if the template(s) from the banner module could be optimized in this way.

I hope I didn't piss you off too much with that. if you do, just tell me and I won't ask any more questions.

greetings, grashalm4u
